web/src/main/java/com/epam/esm/web/config/WebInitializer.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.esm.web.config;
+
+import com.epam.esm.dao.config.DBConfig;
+import com.epam.esm.service.config.ServiceConfig;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+
+public class WebInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {

```

Comment:
I wondering how Web configuration knows where to find declared beans for web layer? (For example controllers)

Reply:
components scan added